onboarding-workflow/src/eversdk/client.ts

```diff
@@ -2,6 +2,10 @@ import { TonClient } from 'npm:@eversdk/core'
 // @deno-types="../../node_modules/@eversdk/lib-web/index.d.ts"
 import { libWeb, libWebSetup } from '../../node_modules/@eversdk/lib-web/index.js'
 
+// https://app.gosh.sh/envs.json
+export const SYSTEM_CONTRACT_ADDR =
```

Comment:
Every gosh version has it's own system contract address, so should it be a constant?